### ISSUE TITLE
About training data of the segmentation task

### ISSUE THREAD

**Issue by JiehongLin** [dingdingcai/OVE6D-pose#9]
Thank you for your great work of OVE6D! 

I have a problem about the training data of the segmentation task. It seems that your employed segmentation networks are trained or finetuned on the target BOP data?

**Comment by dingdingcai** [comment]
You're right. For evaluation, we used the PBR data to train the MaskRCNN to obtain the masks.

**Comment by JiehongLin** [comment]
Ok~ Thank you for your reply!
